lib/wasi/src/state/mod.rs

```diff
@@ -1428,16 +1572,18 @@ impl WasiState {
 
     /// Turn the WasiState into bytes
     pub fn freeze(&self) -> Option<Vec<u8>> {
-        bincode::serialize(self).ok()
+        todo!("temporarily disabled")
+        //bincode::serialize(self).ok()
```

Comment:
Why? cc @syrusakbary

Reply:
We can revert and make bincode an optional dependency if `enable-serde` feature is set